evm/db/backends/level.py

```diff
@@ -0,0 +1,64 @@
+import shutil
+from .base import (
+    BaseDB,
+)
+
+
+class LevelDB(BaseDB):
+    db = None
+    db_path = None
+
+    # Creates db as a class variable to avoid level db lock error
+    def __init__(self, db_path=None):
+        if not db_path:
+            raise TypeError("Please specifiy a valid path for your database.")
+        try:
+            import leveldb
+        except ImportError:
+            raise ImportError("LevelDB requires the leveldb \
+                               library which is not available for import.")
+        if not LevelDB.db:
```

Comment:
How come you're checking the class for the DB path here?  This seems like it should be `self.db`